app/models/vacols/case_docket.rb

```diff
@@ -472,4 +472,27 @@ def self.distribute_appeals(query, judge, dry_run)
   def self.use_by_docket_date?
     FeatureToggle.enabled?(:acd_distribute_by_docket_date, user: RequestStore.store[:current_user])
   end
+
+  def self.ineligible_judges
+    ineligible_vacols_judges.concat ineligible_caseflow_judges
+  end
+
+  def self.ineligible_vacols_judges
+    VACOLS::Staff.where("(STAFF.SVLJ IS NOT NULL OR STAFF.SATTYID IS NOT NULL) AND ((STAFF.SACTIVE = 'I') OR
+     (STAFF.SVLJ <> 'A' AND STAFF.SVLJ <> 'J'))").map do |staff|
+      { sattyid: staff.sattyid, sdomainid: staff.sdomainid, svlj: staff.svlj }
+    end
+  end
+
+  def self.ineligible_caseflow_judges
+    User.joins("LEFT JOIN organizations_users ON users.id = organizations_users.user_id")
+      .joins("LEFT JOIN organizations ON organizations_users.organization_id = organizations.id")
+      .where("users.status != ? OR (organizations_users.admin = '1' AND organizations.type = 'JudgeTeam' AND
+         organizations.status <> 'active') OR (users.id NOT IN (#{admin_users_of_judge_teams}))", "active")
+      .map { |user| { id: user.id, css_id: user.css_id } }
+  end
+
+  def self.admin_users_of_judge_teams
```

Comment:
@calvincostaBAH  you can change this method name and check for non_admins of JudgeTeam
suggestion 👇🏽 
```suggestion
  def self.non_admin_users_of_judge_teams
    organization_ids = JudgeTeam.all.map(&:non_admins).flatten.map(&:id).join(", ")
    OrganizationsUser.where(oganization_id: organization_ids).pluck(:user_id)
  end
```

Reply:
line 496 returns the user ids already instead of the organization id, so wont need the following line. will change name and update to non_admins

Reply:
Updated the suggestion